OrbitClientModel/include/OrbitClientModel/SamplingDataPostProcessor.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CLIENT_MODEL_SAMPLING_DATA_POST_PROCESSOR_H_
+#define ORBIT_CLIENT_MODEL_SAMPLING_DATA_POST_PROCESSOR_H_
+
+#include "OrbitClientData/CallstackData.h"
+#include "OrbitClientModel/CaptureData.h"
+
+namespace sampling_data_post_processor {
+PostProcessedSamplingData CreatePostProcessedSamplingData(const CallstackData& callstack_data,
+                                                          const CaptureData& capture_data,
+                                                          bool generate_summary = true);
+};  // namespace sampling_data_post_processor
```

Comment:
Shouldn't the name of the namespace be `orbit_client_model`?

Reply:
That namespace was more intended just to have the free function properly named.
We haven't really started adding namespaces. If we would have proper namespaces, I would use orbit_client_model::sampling_data_post_processor here. So I'd argue to keep it for now, and wait until we go for namespaces in general.

Reply:
I think fixing the namespaces was one of the goals for the fixit week, so imo it's a good time to start using proper namespace names.

Reply:
Done!